lineman/app/services/form_service.coffee

```diff
@@ -1,6 +1,14 @@
-angular.module('loomioApp').factory 'FormService', ($rootScope, FlashService) ->
+angular.module('loomioApp').factory 'FormService', ($rootScope, FlashService, $filter) ->
   new class FormService
 
+    confirmDiscardChanges: (event, record) ->
+      if record.isModified()
+        console.log record.modifiedAttributes()
+        if !confirm($filter('translate')('common.confirm_discard_changes'))
```

Comment:
I was thinking it felt a little clunky to have entirely new modals for things like deactivate group, deactivate user, remove coordinator, etc. I wonder if we could use this pattern in those places too? Would probably feel better than spinning up a whole new modal here, and I'm thinking the behaviour should be the same for this thing and those things.